### ISSUE TITLE
Wrong COPYING document

### ISSUE THREAD

**Issue by danny-loh** [home-sweet-gnome/dash-to-panel#857]
The License in COPYING document is "GPL v2", but not "GPL v2 or later" referenced in license header of source files or README
https://github.com/home-sweet-gnome/dash-to-panel/blob/25cc6bcd3bb182e541ef00d0c2e0cf2c406113a7/README.md#L181

**Comment by jderose9** [comment]
Hi @danny-loh!

At this time, it is intended for the current and all previous versions of dash-to-panel to be licensed under GPLv2, while allowing copiers to distribute the application with v2 or later version of the GPL license.

The COPYING document includes the text for the GPLv2 license [from the GNU website](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html), as well as instructions on how to apply the license to a particular program.

https://github.com/home-sweet-gnome/dash-to-panel/blob/25cc6bcd3bb182e541ef00d0c2e0cf2c406113a7/COPYING#L242-L248

https://github.com/home-sweet-gnome/dash-to-panel/blob/25cc6bcd3bb182e541ef00d0c2e0cf2c406113a7/COPYING#L288-L291

https://github.com/home-sweet-gnome/dash-to-panel/blob/25cc6bcd3bb182e541ef00d0c2e0cf2c406113a7/COPYING#L296-L299

In other words, the COPYING document contains a copy of official GPLv2 license terms, but it's actually the headers in the source control files that specify which license applies and which licenses copiers may use ("GPL-v2 or later").

I'm not aware of any other GNU licensing documents that are used in the "or later" scenario, so I don't think any modifications to COPYING are necessary. If you know any conventions that are normally used to clarify this scenario, please let me know. Thank you!

**Comment by danny-loh** [comment]
Oh I see. I thought it wrong that there was a "GPL v2 or later" license document.

Thanks.
